maestrowf/interfaces/script/_flux/flux0_26_0.py

```diff
@@ -21,14 +21,45 @@ class FluxInterface_0260(FluxInterface):
 
     flux_handle = None
 
+    @classmethod
+    def connect_to_flux(cls):
+        if not cls.flux_handle:
+            cls.flux_handle = flux.Flux()
+            LOGGER.debug("New Flux handle created.")
+            broker_version = cls.flux_handle.attr_get("version")
```

Comment:
@SteVwonder -- Curiosity question, do all Flux versions support this or is this new? If so, maybe we should promote this to a higher order method and construct the handle in the front end class?

Reply:
It was added in this PR (https://github.com/flux-framework/flux-core/pull/1817) three years ago.  Thus it is in v0.10.0+.  So I think it is safe to promote this as you suggest.